tests/chainer_tests/test_link.py

```diff
@@ -292,6 +295,23 @@ def test_serialize_deserialize_to_uninitialized_param(self):
         self.assertIsInstance(l.x.data, numpy.ndarray)
         numpy.testing.assert_array_equal(l.x.data, ret)
 
+    def test_enable_update(self):
+        self.link.enable_update()
+        self.assertTrue(self.link.x.update_rule.enabled)
+        self.assertTrue(self.link.u.update_rule.enabled)
+
+    def test_disable_update(self):
+        self.link.disable_update()
+        self.assertFalse(self.link.x.update_rule.enabled)
+        self.assertFalse(self.link.u.update_rule.enabled)
+
+    def test_update_enabled(self):
+        self.assertTrue(self.link.update_enabled)
+        self.link.disable_update()
+        self.assertFalse(self.link.update_enabled)
+        self.link.enable_update()
+        self.assertTrue(self.link.update_enabled)
+
```

Comment:
Only a portion of `update_rule`s are enabled at the initial state and so I thought the first `assertTrue` does that test. Isn't that enough?